test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = fixed_map(%{points: integer(1..5)})
+
+      metas_map =
+        fixed_map(%{
+          metas: list_of(points_map, length: 1)
+        })
+
+      user = nonempty(map_of(string(:alphanumeric), metas_map))
+      [user: user]
+    end
+
+    property "winning value is a list or a integer", %{user: user} do
+      check all(users <- user) do
+        {_event, winner} = PointingParty.VoteCalculator.calculate_votes(users)
+        assert is_list(winner) || is_integer(winner)
+      end
+    end
+
+    property "tie when winning value is a list, winner when winning value is an integer", %{
+      user: user
+    } do
+      check all(users <- user) do
+        {event, winner} = PointingParty.VoteCalculator.calculate_votes(users)
+
+        cond do
+          is_list(winner) ->
+            assert event == "tie"
+
+          is_integer(winner) ->
+            assert event == "winner"
+        end
+      end
+    end
+
+    property "the winning value is not more than the highest point value", %{user: user} do
+      check all(users <- user) do
+        {_event, winner} = PointingParty.VoteCalculator.calculate_votes(users)
+
+        max_vote =
+          users
+          |> Enum.map(fn {_username, %{metas: [%{points: points}]}} -> points end)
+          |> Enum.max()
+
+        cond do
+          is_list(winner) ->
+            winner <= max_vote
+
+          is_integer(winner) ->
+            winner <= max_vote
+        end
```

Comment:
This section of code is going always going to pass because it doesn't assert anything. Once you add `assert` to line 52, the test will fail.

Reply:
good catch! fixed.